### PR TITLE
FIP-83: La Tribu (hiring devs from DAO)

### PR DESCRIPTION
**Pull request by eswak**
Forum: https://tribe.fei.money/t/fip-83-la-tribu-hiring-devs-from-a-dao-like-structure/3956 
Snapshot: https://snapshot.org/#/fei.eth/proposal/0xbad3be26475a4fc0acabf4fbea0fa86666b0b3706b0abb15ab904856700aac01

To be merged after #551 and #552

### CODE REVIEW

**Comment by ElliotFriedman** [comment]
LGTM!

File: proposals/dao/fip_83.ts
Comment (thomas-waite) [inline proposals/dao/fip_83.ts:33]: Why are we setting this up so that the beneficiary is `eswak`?

I understand that the plan is then for you to distribute funds to the hired devs, but why don't we first hire the devs and then create timelocks for them where they are the beneficiary?

**Review by thomas-waite — COMMENTED** [review]
The code itself looks good, but a question/thought about process. Why aren't we hiring the devs first and then setting up timelocks for the individual devs once they are hired?

At the moment, when these contracts are deployed then vesting will start immediately I believe. But those devs likely won't have been hired (or at least if one of them has been hired, it's very unlikely that the other will also have been hired at the same time). I feel that the devs should have their own compensation timelocks granted by the DAO

File: proposals/dao/fip_83.ts
Comment (thomas-waite) [reply proposals/dao/fip_83.ts:33]: At the moment, this would require `eswak` to distribute funds to the devs. Why not create individual timelocks for the devs at a later date?

Comment (eswak) [reply proposals/dao/fip_83.ts:33]: it's hard to leave a stable situation & accept the hire if you're not even sure the funding is secured on the other side, this is kind of a chicken & egg problem

also I'm likely to make the FEI>EUR conversion myself & just pay them € with a normal contract through my company. If over time they feel confident going fully on-chain, I can change the timelock beneficiary to a smart contract that allows them to claim their respective tokens eventually

for the ~2-3 months where vesting starts but I didn't find anyone yet... it sucks we have to rely on trust but I'm happy to be fired if you see me cheating. These timelocks vest less than my own paycheck anyway & me onboarding new devs to the DAO kinda need a bit of trust anyway

Comment (thomas-waite) [reply proposals/dao/fip_83.ts:33]: Fair enough. I'd prefer a trust minimalised solution but this arrangement seems like it will necessarily require some trust + it has passed snapshot

**Review by thomas-waite — APPROVED** [review]

File: proposals/dao/fip_83.ts
Comment (thomas-waite) [reply proposals/dao/fip_83.ts:33]: Tagging @ditchfieldcaleb  and @Joeysantoro in case anyone else wants to review

Comment (ElliotFriedman) [reply proposals/dao/fip_83.ts:33]: Let's trust @eswak to do the right thing here. His rep is way more valuable than this drip.

**Comment by ElliotFriedman** [comment]
@ditchfieldcaleb This PR did not require any approval to merge according to the new git bot that was just set up
Screen Shot 2022-02-28 at 9 19 21 PM
. I think that this type of PR should require reviews before merging.

**Review by thomas-waite — APPROVED** [review]

**Comment by ditchfieldcaleb** [comment]
> @ditchfieldcaleb This PR did not require any approval to merge according to the new git bot that was just set up Screen Shot 2022-02-28 at 9 19 21 PM . I think that this type of PR should require reviews before merging.

After merging develop into this branch it should require reviews/approvals

**Comment by ditchfieldcaleb** [comment]
> > @ditchfieldcaleb This PR did not require any approval to merge according to the new git bot that was just set up Screen Shot 2022-02-28 at 9 19 21 PM . I think that this type of PR should require reviews before merging.
> 
> After merging develop into this branch it should require reviews/approvals

Fixed, just had to change base branch to something else and then back again for it to see it, since this PR was created before CODEOWNERS existed.

**Review by ditchfieldcaleb — APPROVED** [review]

**Review by thomas-waite — APPROVED** [review]
LGTM

**Pull request merged**
